src/game/etj_custom_map_votes.cpp

```diff
@@ -243,3 +446,34 @@ bool CustomMapVotes::isValidMap(const std::string &mapName) {
   return G_MapExists(mapName.c_str()) && mapName != level.rawmapname &&
          !MapStatistics::isBlockedMap(mapName);
 }
+
+bool CustomMapVotes::writeCustomvoteFile(const std::string &file,
```

Comment:
I'd probably actually just implement a simple generic JSON interface e.g. 
JSON::readFile(file) and JSON::writeFile(file, root) 
This was probably done this way previously but above would make more sense as there's nothing specific to custom map votes

Reply:
Yeah this would actually make more sense, will implement some basic thing that we can expand later.

Reply:
Added a very simple `JsonUtils` class that simply has these read/write funcs in it, can be expanded later when needed. Should probably refactor the rest of our json stuff into this too but that can be done later.

Reply:
Nice